include/swift/AST/DiagnosticsSema.def

```diff
@@ -2920,7 +2923,9 @@ ERROR(nonclass_convenience_init,none,
 ERROR(cfclass_convenience_init,none,
       "convenience initializers are not supported in extensions of CF types",
       ())
-
+ERROR(cfclass_designated_init_in_extension,none,
```

Comment:
Could you add this error below `designated_init_in_extension` since they're related?

Reply:
Done